spoke-pm2.config.js.template

```diff
@@ -0,0 +1,140 @@
+const env_production = {
+  NODE_ENV:'production',
+  AWS_ACCESS_KEY_ID:'',
+  AWS_SECRET_ACCESS_KEY:'',
+  APOLLO_OPTICS_KEY:'',
+  DEV_APP_PORT:8090,
+  OUTPUT_DIR:'./build',
+  PUBLIC_DIR:'./build/client',
+  ASSETS_DIR:'./build/client/assets',
+  ASSETS_MAP_FILE:'assets.json',
+  CAMPAIGN_ID:'campaign-id-hash',
+  DB_HOST:'127.0.0.1',
+  DB_PORT:28015,
+  DB_NAME:'spokedev',
+  DB_KEY:'',
+  JOB_DB_HOST:'127.0.0.1',
+  JOB_DB_PORT:6379,
+  AUTH0_LOGIN_CALLBACK:'http://example.com:8090/login-callback',
+  AUTH0_LOGOUT_CALLBACK:'http://example.com:8090/logout-callback',
+  AUTH0_DOMAIN:'example.auth0.com',
+  AUTH0_CLIENT_ID:'',
+  AUTH0_CLIENT_SECRET:'',
+  AUTH0_LOGOUT_CALLBACK:'',
+  SESSION_SECRET:'',
+  NEXMO_API_KEY:'',
+  NEXMO_API_SECRET:'',
+  TWILIO_API_KEY:'',
+  TWILIO_APPLICATION_SID:'',
+  TWILIO_AUTH_TOKEN:'',
+  TWILIO_STATUS_CALLBACK_URL:'http://example.com:3000/twilio',
+  SSH_TUNNEL_TARGET:'exampleuser@example.com',
+  SSH_TUNNEL_FORWARDS:'3000:127.0.0.1:6379',
+  MAILGUN_SECRET_KEY:'',
+  MAILGUN_DOMAIN:'',
+  MAILGUN_FROM_EMAIL:'',
+  ROLLBAR_CLIENT_TOKEN:'',
+  ROLLBAR_ACCESS_TOKEN:'',
+  ROLLBAR_ENDPOINT:'https://api.rollbar.com/api/1/item/'
+}
+
+module.exports = {
+  /**
+   * Application configuration section
+   * http://pm2.keymetrics.io/docs/usage/application-declaration/
+   */
+  apps : [
+
+    // First application
+    {
+      name      : 'spoke-server',
+      script    : './build/server/server',
+      env: {
+        COMMON_VARIABLE: 'true'
```

Comment:
I believe that's just example config, not actually used for anything. `env` is just environment variables that get loaded regardless of the `--env` flag, so this is passing an environment variable named `COMMON_VARIABLE` to node, which isn't being used.